### ISSUE TITLE
FCOS with coco Error

### ISSUE THREAD

**Issue by as754770178** [open-mmlab/mmdetection#3340]
I run `fcos_r50_caffe_fpn_4x4_1x_coco.py`, and load the pretrained_model `resnet50_caffe-788b5fa3.pth`, report:
```
unexpected key in source state_dict: conv1.bias, layer1.0.conv1.bias, layer1.0.conv2.bias, layer1.0.conv3.bias, layer1.0.downsample.0.bias, layer1.1.conv1.bias, layer1.1.conv2.bias, layer1.1.conv3.bias, layer1.2.conv1.bias, layer1.2.conv2.bias, layer1.2.conv3.bias, layer2.0.conv1.bias, layer2.0.conv2.bias, layer2.0.conv3.bias, layer2.0.downsample.0.bias, layer2.1.conv1.bias, layer2.1.conv2.bias, layer2.1.conv3.bias, layer2.2.conv1.bias, layer2.2.conv2.bias, layer2.2.conv3.bias, layer2.3.conv1.bias, layer2.3.conv2.bias, layer2.3.conv3.bias, layer3.0.conv1.bias, layer3.0.conv2.bias, layer3.0.conv3.bias, layer3.0.downsample.0.bias, layer3.1.conv1.bias, layer3.1.conv2.bias, layer3.1.conv3.bias, layer3.2.conv1.bias, layer3.2.conv2.bias, layer3.2.conv3.bias, layer3.3.conv1.bias, layer3.3.conv2.bias, layer3.3.conv3.bias, layer3.4.conv1.bias, layer3.4.conv2.bias, layer3.4.conv3.bias, layer3.5.conv1.bias, layer3.5.conv2.bias, layer3.5.conv3.bias, layer4.0.conv1.bias, layer4.0.conv2.bias, layer4.0.conv3.bias, layer4.0.downsample.0.bias, layer4.1.conv1.bias, layer4.1.conv2.bias, layer4.1.conv3.bias, layer4.2.conv1.bias, layer4.2.conv2.bias, layer4.2.conv3.bias
```
But loss is nan
```
2020-07-18 15:25:16,774 - mmdet - INFO - Epoch [1][50/29317]	lr: 3.333e-03, eta: 2 days, 10:47:54, time: 0.602, data_time: 0.070, memory: 5117, loss_cls: nan, loss_bbox: nan, loss_centerness: nan, loss: nan, grad_norm: nan
```

**Comment by hellock** [comment]
Please follow the `Error report` issue template.

**Comment by lengxia** [comment]
i get this error,set lr = 5e-4 can solve it. 
but  low mAP

**Comment by wxhqw** [comment]
@lengxia 
have u solved this problem?

**Comment by lengxia** [comment]
你改一下lr就好了，改小一点

**Comment by as754770178** [comment]
@lengxia 感谢，请教一下，你在COCO数据集上，复现指标了吗？

**Comment by lengxia** [comment]
36.8

**Comment by as754770178** [comment]
`configs/fcos/fcos_x101_64x4d_fpn_gn-head_mstrain_640-800_4x2_2x_coco.py`，学习率调整为0.005，效果复现
